src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -610,16 +634,25 @@ function updateConstSumMessageQn(qnNum) {
 
     if (distributeToRecipients) {
         const $constSumMessageElement = $(`#constSumMessage-${qnNum}-${numOptions - 1}`);
+        const $constSumMinMessageElement = $(`#constSumMinMessage-${qnNum}-${numOptions - 1}`);
```

Comment:
Extract the changes outside the `if` condition seems u are doing the same thing.